src/main/java/frc/robot/lib/bindings/DriveBindings.kt

```diff
@@ -10,6 +13,18 @@ import frc.robot.subsystems.drivetrain.Chassis
 private val DRIVE_LAYOUT =
     DriveLayout.SNAPPING.also { SmartDashboard.putString("drive_layout", it.name) }
 
+val DRIVE_MODIFIERS =
+    DriveModifiers(triggerAdjust = true, distanceSlowing = true).also {
+        SmartDashboard.putBoolean("trigger_adjust", it.triggerAdjust)
+        SmartDashboard.putBoolean("distance_slowing", it.distanceSlowing)
+    }
+
+/**
+ * Stores values for whether to apply trigger speed, distance slowing, and any other future speed
+ * modifiers
+ */
+class DriveModifiers(val triggerAdjust: Boolean = false, val distanceSlowing: Boolean = false)
```

Comment:
Not sure I understand the point of this. Why not just have individual variables for trigger adjust and distance slowing?